cmd/cdi-controller/controller.go

```diff
@@ -145,7 +145,14 @@ func start(ctx context.Context, cfg *rest.Config) {
 		klog.Fatalf("Error building extClient: %s", err.Error())
 	}
 
-	mgr, err := manager.New(config.GetConfigOrDie(), manager.Options{})
+	opts := manager.Options{
+		LeaderElection:             true,
+		LeaderElectionNamespace:    namespace,
+		LeaderElectionID:           "cdi-controller-leader-election-helper",
+		LeaderElectionResourceLock: "leases",
```

Comment:
Should this ResourceLock be configmapsleases as well? We have a configmap for that right now.

Reply:
no we still get the old configmap lock first

Reply:
Nevermind we get the old configmap lock before calling start which contains this code.